### DISCUSSION TITLE
Http Client?

### DISCUSSION THREAD

**Discussion by n0nag0n** [mako-framework/framework#308] [General]
Hey there

I see you have an HTTP Client on an older version of Mako, https://makoframework.com/docs/3.6/learn-more:rest-client just curious if you kept something separately running or if you use a different package with your typical projects? I avoid Guzzle when I can cause it's such a massive package haha

**Comment by freost** [comment]
I removed it a long time ago since it was lacking in features and there are quite a few solid options avaiable. We usually end up using Guzzle in our projects since there's always at least one third party package that requires it anyway.
